### PR TITLE
Fixes: #{2391}

### PR DESCRIPTION
**Pull request by siddhant2002**
### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: src/main/java/com/thealgorithms/strings/Anagrams.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+         * 2nd approach Time Complexity : O(n)
+         * Auxiliary Space Complexity : O(1)
+         * 3rd approach Time Complexity : O(n)
+         * Auxiliary Space Complexity : O(1)
+         * 4th approach Time Complexity : O(n)
+         * Auxiliary Space Complexity : O(n)
+         */
+    }
+
+    void approach1(String s, String t)
```
Comment (siriak) [inline src/main/java/com/thealgorithms/strings/Anagrams.java:40]: Please return a boolean value from the functions instead of printing to console.

**Review by siriak — CHANGES_REQUESTED** [review]
Looks good, only 1 issue needs to be fixed

File: src/main/java/com/thealgorithms/strings/Anagrams.java (hunk 1 above)
Comment (siddhant2002) [reply src/main/java/com/thealgorithms/strings/Anagrams.java:40]: ok. Thanks

**Review by siriak — APPROVED** [review]
LGTM, thanks!

**Pull request merged**